data-seeding/data-seeding.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.0-preview1-27122" />
```

Comment:
@anpete These are from https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json. Any suggestion? Should we leave it floating, add the source to the solution?

Reply:
Floating. Does this have any effect on the other projects?